UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Services/NMDataService.cs

```diff
@@ -382,27 +382,27 @@ public async Task<byte[]> DownloadFssFileAsync(string batchId, string fileName,
 
         }
 
-        public async Task<byte[]> DownloadFSSZipFileAsync(string batchId, string fileName, string correlationId)
+        public async Task<byte[]> DownloadFSSZipFileAsync(string batchId, string fileName, bool isDistributorUser, string correlationId)
         {
             try
             {
-                _logger.LogInformation(EventIds.GetDailyZipNMFileStarted.ToEventId(), "Maritime safety information request to get daily zip NM file started for batchId:{batchId} and fileName:{fileName} with _X-Correlation-ID:{correlationId}", batchId, fileName, correlationId);
+                _logger.LogInformation(EventIds.GetNMZipFileStarted.ToEventId(), "Maritime safety information request to get NM zip file started for batchId:{batchId} and fileName:{fileName} with _X-Correlation-ID:{correlationId}", batchId, fileName, correlationId);
 
-                string accessToken = await _authFssTokenProvider.GenerateADAccessToken(_userService.IsDistributorUser, correlationId);
+                string accessToken = await _authFssTokenProvider.GenerateADAccessToken(isDistributorUser, correlationId);
```

Comment:
As above, please revert this change as well.
```suggestion
                string accessToken = await _authFssTokenProvider.GenerateADAccessToken(_userService.IsDistributorUser, correlationId);
```

Reply:
Done